package.json

```diff
@@ -7,7 +7,8 @@
   "scripts": {
     "build": "cpx index.d.ts dist/ && tsc -p .",
     "build:watch": "cpx index.d.ts dist/ && tsc -p . -w",
```

Comment:
ran into several issues where I'd like to clean the node_modules inside tests also, so I'm sure that there's no old files left-over